### DISCUSSION TITLE
Using BlazorApplicationInsights with Blazorise

### DISCUSSION THREAD

**Discussion by garethjwelsh** [IvanJosipovic/BlazorApplicationInsights#146] [Q&A] [answered]
Hi all,

Has anyone had expereince using Blazorise with this Nuget package. We have managed to use them well together, but we are getting an intermittent fault in our front end code as seen below.

<img width="380" alt="MicrosoftTeams-image" src="https://user-images.githubusercontent.com/12817112/179047871-023d08af-46d9-4511-81a4-8bcd2d551b44.png">

We aren't actually using JSInterop anywhere and the particular issue is intermittent. I know there was a similar fault between this package and Radzen [https://stackoverflow.com/questions/72275562/try-to-track-down-blazor-client-side-exception](url) but wondered if there was a similar issue with Blazorise.

Any insight appreciated!

**Comment by IvanJosipovic** [comment, marked as answer]
Hey,

It's impossible to troubleshoot this with the data you provided. It could be Blazorize or BlazorApplicationInsights. Can you please create and issue with code that can reproduce this issue?
